spec/e2e/has_one_spec.rb

```diff
@@ -206,6 +208,41 @@
       manager.subordinates << employee
       expect(employee.manager(rel_length: 1)).to eq(manager)
     end
+
+    # since chainable: true is an option, this test both checks to see that default options
+    # are honored, and checks to make sure the provided options are merged into the default options
+    it 'honors default options' do
+      expect(
+        employee.manager(chainable: true)
+          .instance_variable_get('@query_proxy')
+          .instance_variable_get('@association_labels')
```

Comment:
Again on these, would like to investigate the possibility of checking the Cypher query.  It might be a bit hacky, but perhaps less hacky then getting instance variables since that spoils encapsulation, I think